models/src/main/kotlin/io/provenance/api/models/p8e/contracts/VOSmartContractLibraryClientCall.kt

```diff
@@ -0,0 +1,7 @@
+package io.provenance.api.models.p8e.contracts
+
+data class VOSmartContractLibraryClientCall(
+    val methodName: String,
+    val parameterClassName: String,
+    val parameterClassJson: String
```

Comment:
I think having a String primitive will make forming this call very difficult. Cant we break this down more with data classes?